### ISSUE TITLE
Project maintenance status?

### ISSUE THREAD

**Issue by rusty-snake** [orgzly/orgzly-android#1039]
There seems to be no developer activity since November 2022.

**Comment by DanielKonopka** [comment]
Maybe it would be wise to transfer maintenance to somebody else? Just a hint. The app is so great it would be a blasphemy to leave it like this.

**Comment by Itrekr** [comment]
If anyone has any well working alternative please let me know because I am not sure if this project is still maintained.

**Comment by alokehpdev** [comment]
Yeah someone else maintaining it would be awesome since there are a lot of forks that aren't merged. If someone could take it up, this app could be so much valuable.

**Comment by idlip** [comment]
I know this app is still useful and has intuitive modern UI.

I guess nobody has talked about it, but now, We have official emacs app for android.
For non-emacs user who only depended on orgzly as the powerhouse, this might not be the good news.

You can get emacs from fdroid (does not support gnutls, which means you cannot use `eww` or add melpa...), but there is sourceforge where they release apk for every commit and has full support to gnutls (and maybe others)

I recommend you guys to give it a try, there is a dedicated section for `Android` in `info-emacs-manual` for you to read.

I reckon, it can be almost (if not, 100%) THE org-mode reader with full experience.

Coming to the main aspect, org-agenda, it can display as usual, I have not yet figured if it can send notifications for tasks scheduled (need to explore a bit more)

Give it a go, and spread the word: https://www.emacswiki.org/emacs/EmacsOnAndroid

**Comment by markamaze** [comment]
I agree it would be a shame if this project dies. I've used it for years and explored the code as I've been learning. I've got an idea for addressing an issue but if pr's aren't going anywhere I won't bother. Totally understandable if @nevenz needs a break, but perhaps other experienced contributors could have access to keep development going. I'd love to start using it for some contributor credit.

**Comment by Itrekr** [comment]
> I agree it would be a shame if this project dies. I've used it for years and explored the code as I've been learning. I've got an idea for addressing an issue but if pr's aren't going anywhere I won't bother. Totally understandable if @nevenz needs a break, but perhaps other experienced contributors could have access to keep development going. I'd love to start using it for some contributor credit.

Would it possible to fork the whole project and get a group of new maintainers together to work on it?

**Comment by YesNoMaybeItDepends** [comment]
> Would it possible to fork the whole project and get a group of new maintainers together to work on it?

The license certainly allows it, just needs someone to take the initiative and to spread the word, both to get a group of maintainers and to share the fork with the public. 

Should probably reach out to @nevenz first though, don't know if anyone else already tried but I'll see if I can get in contact with him.

**Comment by xeruf** [comment]
Yeah, the project is already hosted in an organization, so if a few people step up he could add them to it so it lives on in the original repo. With a fork we loose all issues and stuff.

**Comment by Itrekr** [comment]
> Yeah, the project is already hosted in an organization, so if a few people step up he could add them to it so it lives on in the original repo. With a fork we loose all issues and stuff.

That would be best, but we don't know if the project is abandoned or not.

**Comment by danielkrajnik** [comment]
It's way too soon for Orgzly to die. We still haven't solved the infamous #287

**Comment by alensiljak** [comment]
> Would it possible to fork the whole project and get a group of new maintainers together to work on it?

Isn't that the point of open source? It's been years and yet nobody has really bitten the bullet to actually do that yet.

**Comment by Itrekr** [comment]
> > Would it possible to fork the whole project and get a group of new maintainers together to work on it?
> 
> Isn't that the point of open source? It's been years and yet nobody has really bitten the bullet to actually do that yet.

I know. It was more of a: "Do we have the people to pull this of?" Not a: "Are we allowed to do this?"

**Comment by amberin** [comment]
@Itrekr I believe it is at least worth putting a group of maintainers together and making an attempt.

@YesNoMaybeItDepends Did you ever manage to reach @nevenz?

**Comment by markamaze** [comment]
> @Itrekr I believe it is at least worth putting a group of maintainers together and making an attempt.
> 

I'd be willing to commit to it, whether forked or access is granted to orgzly. Have been studying and practicing software development for a while and want hands on experience. Couldn't lead anything but can take tasks and work on them with minimal guidance. Whatever knowledge I'm missing I'll put in the effort to gain it. 

Any others willing? Perhaps we can gather a list of willing people here.

**Comment by alensiljak** [comment]
I'd be interested in participating and contributing here and there, depending on the time available.

**Comment by amberin** [comment]
There is now a fork: https://github.com/orgzly/orgzly-android/pull/1037#issuecomment-1685698893

**Comment by anotherboringuser** [comment]
Has anyone tried reaching out to @nevenz via the support@orgzly.com email adress? 
The problem with just forking the project is that afaik there’s no way to contact or push updates to the more than 100.000 people who downloaded the app in the Playstore. 
I’m really worried about this project dying but there’s also little I can contribute to it.

**Comment by karlicoss** [comment]
@anotherboringuser yeah, I tried reaching few weeks ago via that email, no response :(

**Comment by Itrekr** [comment]
@anotherboringuser Whilst this is true, active users will find the app if we get it on the F-droid store alongside the original one. Maybe we give it a new name or call it Orgzly Revived or something.

**Comment by erikmd** [comment]
@Itrekr Yes, in this case maybe "Orgzly Reboot"? (like https://github.com/keepassx → https://github.com/keepassxreboot)

**Comment by alensiljak** [comment]
> @Itrekr Yes, in this case maybe "Orgzly Reboot"? (like https://github.com/keepassx → https://github.com/keepassxreboot)

Let's continue that discussion at https://github.com/orgzly-community/orgzly-android/discussions/8

**Comment by stokito** [comment]
In the [PlayStore page](https://play.google.com/store/apps/details?id=com.orgzly&pli=1) the "App support" section has an address and phone. So @amberin Maybe you can contact the authors?

**Comment by alensiljak** [comment]
> In the [PlayStore page](https://play.google.com/store/apps/details?id=com.orgzly&pli=1) the "App support" section has an address and phone. So [@amberin](https://github.com/amberin) Maybe you can contact the authors?

What would be the point of that?

**Comment by xeruf** [comment]
Reviving this organization instead of splitting the community maybe?

**Comment by amberin** [comment]
The Revived fork was only created because nobody has been able to get in touch with @nevenz, the only person with control over this organization.
